src/msgraph-cli.csproj

```diff
@@ -9,6 +9,8 @@
     <PublishSingleFile>true</PublishSingleFile>
     <SelfContained>false</SelfContained>
     <PublishReadyToRun>true</PublishReadyToRun>
+    <AssemblyName>mgc</AssemblyName>
+    <Version>0.1.0-preview1</Version>
```

Comment:
`0.1.0-preview1` should be `0.1.0-preview.1`

https://github.com/microsoftgraph/msgraph-sdk-design/blob/master/Versions.md#version-format-per-language